.github/actions/terraform-observe/opal-validation/action.yaml

```diff
@@ -0,0 +1,71 @@
+name: 'OPAL validation'
+description: 'run tf apply and destroy to validate opal'
+inputs:
+  observe-customer-id:
+    description: "customer id for test observe instance"
+    required: true
+  observe-domain:
+    description: "observeinc.com or observe-staging.com"
```

Comment:
I don't think we should run against prod in any of our testing

Reply:
good idea -- although maybe we want to allow 3rd party devs to have a "testing" instance that is a prod-domain observe instance?

Reply:
I'd prefer that when we get to that point we spin up a small cluster for that. We don't actually need data for testing the OPAL, just an up-to-date Apiserver/transformer

Reply:
cool, i'll keep the input option just in case we change this decision in the future, but to your point i'll just make staging the input value that we use all the time -- that way we can save ourselves the trouble of keeping yet another secret (that isn't even very secret-y to begin with)